packages/core/src/Asset/util/isValidHandle.ts

```diff
@@ -1,4 +1,11 @@
+/**
+ * Alphanumeric: [a-z][A-Z][0-9]
+ * Dash: -
+ * Underscore: _
+ * Period: .
+ * Pipe: | (in general it is not valid, but it exists on mainnet)
```

Comment:
maybe it's worth wording like 'an exception exists in... '

Reply:
https://github.com/input-output-hk/cardano-js-sdk/pull/1089/commits/57126533d688cdbbcd243eca11a12cc0b8869a6b (already squashed)